samples/RenderTest/RenderTest.csproj

```diff
@@ -186,6 +186,11 @@
       <SubType>Designer</SubType>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SkiaSharp">
+      <Version>1.57.1</Version>
```

Comment:
Don't reference packages directly, import the corresponding `props` file from `build` directory